styles.css

```diff
@@ -0,0 +1,674 @@
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+  background-color: #fff;
+  display: block;
+}
+
+.flist {
+  background-color: black;
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  gap: 20px;
+}
+
+.flists {
+  background-color: black;
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 10px 10px;
+  gap: 20px;
+}
+
+.fa-facebook-f {
+  color: #d3d3d3;
+}
+
+.flist a {
+  text-decoration: none;
+  color: #d3d3d3;
+  display: flex;
+  list-style: none;
+}
+
+.bar {
+  color: #d3d3d3;
+  cursor: pointer;
+  padding: 10%;
+}
+
+.first {
+  background-image: url("images/background.png");
+}
+
+.nav {
+  display: none;
+}
+
+.hello {
+  padding: 10px 40px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  overflow-x: hidden;
+}
+
+.hello h1 {
+  font-size: 30px;
+  font-weight: 700;
+  color: #ec5242;
+}
+
+.hello h2 {
+  text-transform: uppercase;
+  font-weight: 900;
+  line-height: 55px;
+  font-family: sans-serif;
+  background-image: linear-gradient(rgba(255, 38, 0, 0.76), rgba(255, 51, 0, 0.575)), url("images/lbackground.jpg");
+  color: transparent;
+  -webkit-background-clip: text;
+  font-size: 45px;
+}
+
+.hello p {
+  background-color: #f5f5f5;
+  border: #d3d3d3 1px solid;
+  padding: 10px;
+  font-size: 17px;
+  line-height: 30px;
+  width: 60%;
+  font-weight: 500;
+  color: #272a31;
+  font-family: "Lato", sans-serif;
+}
```

Comment:
In the mobile view for the headline section please fix the following:

- [ ] Kindly make the hamburger button bigger and make its color darker to match the given design.
- [ ] Please decrease the margin between the two heading texts to match the design.
- [ ] Please increase the width of the text box below the main heading to match the given design, Kindly see the images below to have more understanding.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170760699-14114ac5-49ff-433d-a6fb-0468d5ed0249.png)  | ![image](https://user-images.githubusercontent.com/68796232/170760671-d46ee9aa-cb26-4465-b817-db8b11aea7cc.png)  |